x-pack/plugins/ingest_manager/server/services/epm/packages/remove.ts

```diff
@@ -20,7 +20,7 @@ export async function removeInstallation(options: {
   // TODO:  the epm api should change to /name/version so we don't need to do this
   const [pkgName] = pkgkey.split('-');
   const installation = await getInstallation({ savedObjectsClient, pkgName });
-  const installedObjects = installation?.installed || [];
+  const installedObjects = installation?.installed || { references: [], es_index_patterns: {} };
```

Comment:
Can you change `installedObjects` to point to installation?.installed.references and not change the rest of the code? And perhaps change `installedObjects` to `installedRefs`

Reply:
Yep, I'll change that, thanks.